deploy-lambda/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: Deploy To Environment
+description: Deploy a zip to lambda
+inputs:
+  role-to-assume:
+    description: IAM role
+    required: true
+  lambda-function:
+    description: Lambda function name
+    required: true
+  slack-webhook-url:
+    description: Slack URL to post to
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        role-to-assume: ${{ inputs.role-to-assume }}
+        aws-region: us-east-1 # your AWS region
+    - name: Package the Lambda function
+      run: |
+        zip -r function.zip .
```

Comment:
Based on the [ocs_saver](https://github.com/mbta/ocs_saver/blob/main/build.sh#L36) and (former) [lamp](https://github.com/mbta/lamp/blob/117465e7fec1218a92ee567c5d4c60063222bff0/.github/actions/lambda_deploy/action.yaml#L69) deploy actions, you'll likely need to make parts of this command customizable in order to make it usable for a wide array of use cases. I'm thinking two vars: `path-to-zip` (default: `.`) and `zip-flags` (default: `-r`). You can probably keep the output filename the same, though (especially since you have to refer to it below).

Reply:
done